rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 team leads, one of them must be in @paritytech/locks-review and the other in @paritytech/polkadot-review.
```

Comment:
It seems like there's some disagreement about this requirement and others similar to it. I'll try to decompose the sentence according to my literal interpretation:

> 🔒️ed lines are reviewed by **2** team leads

Emphasis on "2".

> one of them must be in ...

Let's call this agent `DeveloperA`.

> and the other in ...

"the other" literally refers to another agent different from `DeveloperA`. Let's call this `DeveloperB`.

All of the above phrasing to me strongly suggests 2 different agents for this requirement.

Now it might be that someone is in both `@paritytech/locks-review` and `@paritytech/polkadot-review`. https://matrix.to/#/!gJeGMHCcDoIwsHIJri:matrix.parity.io/$5H7WC-VGc4Am5HuvivqpuqA-jR_ic2SmWiz0x4HK1cg raised the question:

> why are you only counting towards one group if you are part of multiple? 

If we'd make it so 1 person can fulfill the criteria by being in both groups, we'd effectively lower the requirement to a single approval (1 person) instead of 2 like I understood by the literal interpretation. If that's intentional, I suggest clarifying the phrasing as e.g.

>🔒️ed requires 1 approval from @paritytech/locks-review and 1 approval from @paritytech/polkadot-review

That removes any implication of "2 team leads" since it only mentions 1 approval from each group, which might be given by the same person if they are in both groups. What's your opinion?

cc @bkchr @TriplEight